### PR TITLE
In case the script snapshotter fails on Dart_Initialize, dump the message.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/tools/sky_snapshot/vm.cc [hunk 1]
```diff
@@ -26,10 +28,15 @@ static const char* kDartArgs[] = {
...
-  CHECK(Dart_Initialize(
-            reinterpret_cast<uint8_t*>(&kDartVmIsolateSnapshotBuffer), nullptr,
-            nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr,
-            nullptr, nullptr, nullptr, nullptr, nullptr) == nullptr);
+  char* init_message = Dart_Initialize(
+      reinterpret_cast<uint8_t*>(&kDartVmIsolateSnapshotBuffer), nullptr,
+      nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr, nullptr,
+      nullptr, nullptr, nullptr, nullptr);
+  if (init_message != nullptr) {
+    std::cerr << "Dart_Initialize Error: " << init_message << std::endl;
```
Comment (jason-simmons) [inline sky/tools/sky_snapshot/vm.cc]: in other places sky_snapshot is using LOG(ERROR)

**Comment by collinjackson** [comment]
lgtm

File: sky/tools/sky_snapshot/vm.cc (hunk 1 above)
Comment (chinmaygarde) [inline sky/tools/sky_snapshot/vm.cc]: Dang, I was trying to be consistent with https://github.com/flutter/engine/blob/master/sky/tools/sky_snapshot/main.cc#L22

**Comment by jason-simmons** [comment]
lgtm

**Pull request merged**